justfile

```diff
@@ -139,9 +139,9 @@ bench_against_main:
 
 # Measure test coverage
 coverage: install_rust_llvm_tools_preview install_cargo_grcov
-    mkdir -p target/coverage
-    # Remove old profiling information and coverage reports
-    rm -rf target/coverage/*
+    # Old build artifacts seem to be able to mess up coverage data (see #508),
```

Comment:
First thought "Could we use `cargo clean`. Second thought: "Ah, target is needed".

Reply:
Actually, it is not needed, as it is recreated when running `cargo test` afterwards (also, any non-existing path in `LLVM_PROFILE_FILE` is automaically created).

My first thought was using `cargo clean` as well, then I thought "why calling a tool for cleaning a directory?". Would you rather have gone with using Cargo?